worker/buildbot_worker/scripts/create_worker.py

```diff
@@ -32,11 +32,11 @@
     import os.path
     basedir = os.path.abspath(os.path.dirname(__file__))
 
-# note: this line is matched against to check that this is a buildslave
+# note: this line is matched against to check that this is a worker
 # directory; do not edit it.
 application = service.Application('buildslave')
```

Comment:
Yes, this should be changed.
I want to get in this PR most of semantically insignificant changes, for which no careful review is needed.

This particular change requires thinking IMO and I would like to do it in separate PR.

@tardyp, also there are bunch of not renamed strings that I think contains errors, I didn't touched them too.